### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by zinabat**
* Adds screenshot and documentation
* Localize the "none" placeholder
* Add Legacy Edge to Sauce

### CODE REVIEW

**Review by tenbradley — APPROVED** [review]
Overall, looks good. The example profile card should be updated once we have everything actually merged.

File: README.md [hunk 1]
```diff
@@ -34,14 +36,30 @@ npm install @brightspace-ui-labs/user-profile-card
...
 
 | Property | Type | Description |
 |--|--|--|
-| | | |
+|online|Boolean|Whether the user is currently online|
+|user-attributes|Array|A list of attributes for the user such as role and pronouns|
+|progress-viewable|Boolean|Whether the current viewer can access the users progress|
+|editable|Boolean|Whether the tagline is editable by the current viewer|
+|tagline|String|The tagling for the user|
+|awards|Boolean|Whether the awards are viewable|
```
Comment (m6jones) [inline README.md:61]: How is this a boolean where do we get the data?

File: README.md [hunk 2]
```diff
@@ -34,14 +36,30 @@ npm install @brightspace-ui-labs/user-profile-card
...
 ```
 
 **Properties:**
 
 | Property | Type | Description |
 |--|--|--|
-| | | |
+|online|Boolean|Whether the user is currently online|
+|user-attributes|Array|A list of attributes for the user such as role and pronouns|
+|progress-viewable|Boolean|Whether the current viewer can access the users progress|
```
Comment (m6jones) [inline README.md:58]: Where do we get the progress?

Comment (m6jones) [reply README.md:58]: I understand now ignore this.

Comment (tenbradley) [reply README.md:58]: progress should be a link to the user's progress page. Should only be viewable by teachers (and maybe learner/parent). Wasn't sure how else to do this.

**Pull request merged**